### DISCUSSION TITLE
Finding dates in page content

### DISCUSSION THREAD

**Discussion by kbfifi** [requarks/wiki#6491] [Error / Bug Report] [answered]
I use wikijs to keep track of issues on my projects. Each issue has a heading like this:
"20-8-2021 DNS no longer working on machine 1"

Is use Postgres DB and Postgres search engine. version 2.5.214 via docker-compose
When I search for "20-8-2021" I hope to find all issues on several pages added with this date. The result is that it finds a lot of pages except the pages I'm looking for. I suspect it is related to dashes that need to be escaped or word boundries.

I would assume that necessary escapes would be handled by WikiJS and not by the user. Any suggestions? Bug, change request or am I missing something?

**Comment by NGPixel** [comment, marked as answer]
This is a limitation of how the search engine works. The text is broken down in tokens so 20-8-2021 wouldn't be searched as complete string.

This is something that can be revisited in v3 to work better with dates, IP addresses and other non-word tokens.

**Reply by kbfifi** [reply]
Thanks for your explanation. Would elasticsearch as search engine not have this problem? Or is it a general limitation for all search engines in WikiJS ?

**Reply by kbfifi** [reply]
I would like to know if it makes sense to invest in elasticsearch
